tools/codegen/api/cpp.py

```diff
@@ -164,7 +165,7 @@ def default_expr(d: str, t: Type) -> str:
     return JIT_TO_CPP_DEFAULT.get(d, d)
 
 # Convert an argument into its C++ API form
-def argument(a: Union[Argument, TensorOptionsArguments, ThisArgument]) -> CppArgument:
+def to_cpp_argument(a: Union[Argument, TensorOptionsArguments, ThisArgument]) -> CppArgument:
```

Comment:
If you're going to change the naming convention here, you should also be changing it in the other api modules, as they all apply consistent naming for how to get to Argument form. But let me try to defend the existing naming convention and see if I can convince you that it is good :)

The current convention is:
* Each API module's functions are implicitly assumed to be operating on the data types of that module. So therefore. cpp.Argument, not cpp.to_cpp_argument (the second cpp is redundant).
* API modules are always accessed using qualified accessor (`from tools.codegen.api import cpp`, never `from tools.codegen.api.cpp import *`)
* Types are always fully qualified within the name (CppArgument, not cpp.Argument)
* If you refer to a type that is not in this module, you have to add it to the function name (e.g., `legacydispatcher_exprs`)

In the first version of this code I explicitly qualified names with things like `cpp_argument`, but it was really wordy, which is why I settled on actually using the namespacing mechanism here.